api/extensions/ext_celery.py

```diff
@@ -160,6 +160,13 @@ def __call__(self, *args: object, **kwargs: object) -> object:
             "task": "schedule.clean_workflow_runlogs_precise.clean_workflow_runlogs_precise",
             "schedule": crontab(minute="0", hour="2"),
         }
+    if dify_config.ENABLE_WORKFLOW_RUN_CLEANUP_TASK:
```

Comment:
it should be added to .env.example and docker-compose.yaml, you can refer to ENABLE_CLEAN_MESSAGES